README.md

```diff
@@ -45,7 +46,7 @@ v.getPalette().then((palette) => console.log(palette))
 The default browser entry for `node-vibrant` has web worker support. You should make sure that `worker-loader` is installed:
 
 ```
-$ npm install --save-dev worker-loader
```

Comment:
Let's revert these. I like being able to easily copy+paste too but there's arguments to be made to make copy+paste commands harder and being able to easily communicate what this line is for at a glance as syntax highlighting for commands isn't easy in markdown

Reply:
Gotcha. 😄

Reply:
Reverted and squashed! I also added `bash` to the last shell code block.